examples/complete-example/README.md

```diff
@@ -39,21 +43,19 @@ certificate and the --resolve option to set the Host header of a request with ``
     
     To get coffee:
     ```
-    $ curl --resolve cafe.example.com:443:$IC_IP https://cafe.example.com/coffee --insecure
+    $ curl --resolve cafe.example.com:$IC_HTTPS_PORT:$IC_IP https://cafe.example.com:$IC_HTTPS_PORT/coffee --insecure
     Server address: 10.12.0.18:80
     Server name: coffee-7586895968-r26zn
     ...
     ```
     If your rather prefer tea:
```

Comment:
Perhaps add the motivation behind these changes in PR description and/or in commit message:

Commit message example:
```
Add IC_HTTPS_PORT in the examples

- [why]
```